src/codegen/llvm/intrin_rule_rocm.cc

```diff
@@ -29,8 +29,10 @@ namespace llvm {
 TVM_REGISTER_GLOBAL("tvm.intrin.rule.rocm.exp")
 .set_body(DispatchExternOCML);
 
-TVM_REGISTER_GLOBAL("tvm.intrin.rule.rocm.fma")
-.set_body(DispatchExternOCML);
+// On AMD GPU, fma is slower than mac
+// removing fma dispatch allows backend to generate faster mac instruction
```

Comment:
We should change this instead to https://github.com/dmlc/tvm/blob/master/src/codegen/llvm/intrin_rule_llvm.cc#L20